### DISCUSSION TITLE
I can't verify my domain

### DISCUSSION THREAD

**Discussion by kaiabound-dev** [WalletConnect/walletconnect-monorepo#5237] [Developer Support] [answered]
<img width="412" alt="image" src="https://github.com/user-attachments/assets/1617bc77-2587-4b2e-a086-ae95c7415dd6">

When I click the left "Verify Domain", Nothing happened.

Also, When I read the guide, there would be a domain verification section in settings (TXT, etc...), but not.

<img width="550" alt="image" src="https://github.com/user-attachments/assets/698f9c84-2748-4d5f-b010-3b9d176a7aec">

also, I found this page, and even I typed in my domain, There is no more actions that I can proceed.

How should I do for domain verification?

**Comment by glitch-txs** [comment]
hello, could you share a video?

**Comment by ngn999** [comment]
@glitch-txs 
Same problem here, now.
And Web3Modal from WalletConnectSwiftV2 use my app bundle id as Origin in http/websocket request header. but my app bundle id is not a valid domain, I can't add it, verify it.
Or anyway can I edit the `allow list`?Some documents mention it.

**Reply by devceline** [reply, marked as answer]
We are removing the verify button, but for now you can verify through the allow list. There will be a verify button where appropriate
